src/handlebars/layout/layout.hbs

```diff
@@ -10,6 +10,7 @@
     <!-- leafletJS stylesheets -->
     <link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.3/dist/leaflet.css" integrity="sha512-07I2e+7D8p6he1SIM+1twR5TIrhUQn9+I6yjqD53JQjFiMf8EtC93ty0/5vJTZGF8aAocvHYNEDJajGdNx1IsQ==" crossorigin=""/>
     <link rel="stylesheet" href="https://unpkg.com/leaflet-geosearch@2.2.0/assets/css/leaflet.css" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
maybe move this to just below the font-awesome stylesheet, since there's a comment indicating that the last two are leafetJS stylesheets